substrate/frame/contracts/uapi/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "pallet-contracts-uapi"
+version = "4.0.0-dev"
+authors.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository.workspace = true
+description = "Exposes all the host functions that a contract can import."
+
+[dependencies]
+paste = { version = "1.0", default-features = false }
```

Comment:
This is unused, right?

Reply:
it's used by the hash_fn  and impl_wrapper_for macro that both need to do some function name concatenation